test/harness/index.js

```diff
@@ -140,7 +151,7 @@ function assert_invalid(bytes) {
 const assert_malformed = assert_invalid;
 
 function assert_soft_invalid(bytes) {
-    test(() => {
+    uniqueTest(() => {
```

Comment:
Consistency with WPT `test` seems like a good enough reason. I prefer having the name first, but I think consistency is a stronger argument than my personal preferences, so I'd leave this as is.